common/rpackagemanager.h

```diff
@@ -74,6 +77,180 @@ class RPackageManager {
 
 };
 
+/**
```

Comment:
Lets put this into either "rsources.{cc,h}" or a new "rsource_deb822.{cc,h}" - the rpackagemanager file is not quite the right place for this (and/or rsources_deb822 and rsources_manager).